docs/ChannelValue.md

```diff
@@ -23,14 +23,21 @@ import { ChannelValue } from '@pxblue/react-native-components';
 | iconSize  | The size to render the icon                         | `number`                               | no       | `fontSize`          |
 | iconColor | A component to render for the icon                  | `string`                               | no       | `theme.colors.text` |
 | units     | Text to display for the units                       | `string`                               | no       |                     |
-| unitSpace | Whether to show a space between the value and units | `'show' \| 'hide' \| 'auto'`           | no       | `'auto'`            |
+| unitSpace | Whether to show a space between the value and units | `'show'` \| `'hide'` \| `'auto'`       | no       | `'auto'`            |
 | prefix    | If true, shows units before the value               | `boolean`                              | no       | `false`             |
 | fontSize  | The size used for the text elements                 | `number`                               | no       | 'medium'            |
 | color     | The color used for the text elements                | `string`                               | no       | `theme.colors.text` |
 | theme     | Theme value overrides                               | `$DeepPartial<ReactNativePaper.Theme>` | no       |                     |
 
 </div>
 
+#### unitSpace Whitelist Values
+
+```tsx
+const prefixUnitWhitelist = ['$'];
+const suffixUnitWhitelist = ['%', '℉', '°F', '℃', '°C', '°'];
+```
+
```

Comment:
If you're going to put this in here, you're going to have to explain what it means. Whitelist is an internal variable name...I don't think anyone will intuitively know what that means in this context.

Maybe put an asterisk on unitSpace and then do a callout >

```suggestion
> Setting unitSpace to auto will show a space for all units except for '%', '℉', '°F', '℃', '°C', and '°'. When `prefix` is true, the space will be shown for all units except for '$'.
```